tips/TIP-0045/tip-0045.md

```diff
@@ -0,0 +1,1646 @@
+      <b>Type</b>
+    </td>
+    <td>
+      <b>Description</b>
+    </td>
+  </tr>
+  <tr>
+    <td>Input Type</td>
+    <td>uint8</td>
+    <td>Set to <strong>value 0</strong> to denote a <i>UTXO Input</i>.</td>
+  </tr>
+  <tr>
+    <td>Transaction ID</td>
+    <td>ByteArray[36]</td>
+    <td>The identifier of the transaction that created the referenced output.</td>
+  </tr>
+  <tr>
+    <td>Transaction Output Index</td>
+    <td>uint16</td>
+    <td>The output index of the referenced output.</td>
+  </tr>
+</table>
+
+### Context Inputs
+
+The `Context Inputs` field holds inputs that provide additional contextual information for the execution of a
+transaction, such as for different functionality related to accounts, commitments, or Mana rewards. Context inputs do
+not need to be unlocked. This TIP defines 3 types of context inputs:
+
+- _Commitment Input_.
+- _Block Issuance Credit Input_.
+- _Reward Input_.
+
+#### Commitment Input
+
+A _Commitment Input_ allows referencing a commitment to a certain slot. It is used to provide a notion of time for
+transaction execution that is linked to the containing _Block_'s _Issuing Time_. It proves that the time at the
+transaction execution is past a certain slot in the past, as the slot has already been committed. The slot reference is
+expressed as the _Commitment ID_ and can be resolved to the _Commitment_ value before executing the transaction. The
+_Commitment_ itself provides the `Slot Index` which serves as the time reference. A Block that contains a transaction
+with a Commitment input has additional validation rules as defined in [TIP-0046](../TIP-0046/tip-0046.md). Only one
+_Commitment Input_ may be present in a transaction.
+
+It is serialized as follows:
+
+<details>
+  <summary>Commitment Input</summary>
+  <blockquote>A Commitment Input allows referencing a commitment to a certain slot and is used to provide a notion of time for transaction execution that is linked to the containing Block's <code>Issuing Time</code>.</blockquote>
+</details>
+<table>
+  <tr>
+    <td>
+      <b>Name</b>
+    </td>
+    <td>
+      <b>Type</b>
+    </td>
+    <td>
+      <b>Description</b>
+    </td>
+  </tr>
+  <tr>
+    <td>Context Input Type</td>
+    <td>uint8</td>
+    <td>Set to <strong>value 0</strong> to denote a <i>Commitment Input</i>.</td>
+  </tr>
+  <tr>
+    <td>Commitment ID</td>
+    <td>ByteArray[36]</td>
+    <td>The commitment identifier to reference to.</td>
+  </tr>
+</table>
+
+#### Block Issuance Credit Input
+
+A _Block Issuance Credit Input_ provides the BIC balance of a specific account as context to transaction execution of a
+specific slot. It is required for any _Account_ transition and destruction if the account contains a _Block Issuer
+Feature_, as any operation on the _Account_ can only be allowed if the BIC balance is non-negative at a specific point
+in time. A _Block Issuance Credit Input_ always requires a _Commitment Input_. The input will be resolved to the BIC
```

Comment:
> A _Block Issuance Credit Input_ always requires a _Commitment Input_.
This doesn't seem to be true, the node accepts BIC input without a commitment input.

Reply:
Do you happen to have an example or a alphanet transaction that shows that?
I don't see how it could get past this check: https://github.com/iotaledger/iota-core/blob/de711606b447c36733ee60d3b72a0932166d07c1/pkg/protocol/engine/ledger/ledger/vm.go#L96.